FHICORC/ViewModels/QrScannerViewModels/ScanSHCVaccineResultViewModel.cs

```diff
@@ -213,5 +237,22 @@ private void CheckControlType()
                 BannerText = "SCANNER_NO_BANNER_TEXT".Translate();
             }
         }
+
+        public void SetAccessibilityTextDate()
+        {
+            try
+            {
+                var vaccinationDate = DateTime.ParseExact(VaccinationDate, "yyyy-MM-dd", CultureInfo.InvariantCulture);
+                var dateOfBirth = DateTime.ParseExact(DateOfBirth, "yyyy-MM-dd", CultureInfo.InvariantCulture);
+                VaccinationDateAccessibilityText = string.Format("{0:dd. MMMM yyyy}", vaccinationDate);
+                DateOfBirthAccessibilityText = string.Format("{0:dd. MMMM yyyy}", dateOfBirth);
```

Comment:
Should there be two individual try-catch blocks here? So if vaccine date fails, it does not automatically fail data of birth as well (and inverse), in case only one of them has an exception :)

Reply:
Yes good point, I will fix it :)